publisher/product/ontology/build.sh

```diff
@@ -101,40 +101,52 @@ function runHygieneTests() {
   #
   # Run consistency-check for DEV and PROD ontologies
   #
-  logRule "consistency-check: run for ..."
 
-  # For now consistency check are turned off.  
+  rm -f "${hygiene_product_tag_root}/consistency-check.log" &>/dev/null
 
-  if false && [ -s "${source_family_root}/${DEV_SPEC}" ] ; then
-    rm -f ${TMPDIR}/{console.txt,ret.txt}
-    logItem " DEV:${DEV_SPEC}" "$(getOntologyIRI < "${source_family_root}/${DEV_SPEC}")"
-    if ${ONTOVIEWER_TOOLKIT_JAVA} --data "${source_family_root}/${DEV_SPEC}" \
-        --output ${TMPDIR}/ret.txt $(test -s "${source_family_root}/catalog-v001.xml" && echo "--ontology-mapping ${source_family_root}/catalog-v001.xml") \
+  test -n "${HYGIENE_SPEC_WARN}" && logRule "run consistency check at level: warning" && \
+  for SPEC in ${HYGIENE_SPEC_WARN} ; do
```

Comment:
The _warning_ level consistency-check test is disabled by default - empty `${HYGIENE_SPEC_WARN}` variable